httpserver-rs/bin/main.rs

```diff
@@ -0,0 +1,86 @@
+//! Http Server implementation for wasmcloud:httpserver
+//!
+//!
+use std::{collections::HashMap, convert::Infallible, sync::Arc};
+use tokio::sync::RwLock;
+use wasmbus_rpc::{core::LinkDefinition, provider::prelude::*};
+use wasmcloud_provider_httpserver::{load_settings, HttpServer};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // handle lattice control messages and forward rpc to the provider dispatch
+    // returns when provider receives a shutdown control message
+    provider_main(HttpServerProvider::default())?;
+
+    eprintln!("HttpServer provider exiting");
```

Comment:
Should this emit to `stderr` ? If this line executes as a result of a graceful exit, we should make it explicit that this isn't a "bad" exit.

Reply:
env_logger and pretty_env_logger both write to stderr by default - for all logs.- I was following that convention.